### DISCUSSION TITLE
Level always visible

### DISCUSSION THREAD

**Discussion by F-Burning** [opentoonz/opentoonz#6451] [Ideas]
A right-click option on the level to make it always visible, no matter how extend is the xsheet or timeline. A visual example is how works keyframes loops.

This can very helpful to image references, for example, but it have so much advantages.

**Comment by gab3d** [comment]
mmmm... maybe this could be a good option to have.
In After Effects, when you drop an image into a timeline it spans for the while lenght of it.
So maybe we could have the opposite as an option...
🤷🏻‍♂️

**Comment by RodneyBaker** [comment]
This request reads as Explicit Holds... as currently seen in Tahoma2D.
@F-Burning Is that the basic idea of this request?

**Comment by RodneyBaker** [comment]
Converting this to a Discussion where additional details can be investigated.
